engine/table/src/main/java/io/deephaven/engine/table/impl/AddOnlyToBlinkTableAdapter.java

```diff
@@ -42,14 +45,17 @@ public static Table toBlink(@NotNull final Table table) {
             return table;
         }
 
+        Table addOnlyTable;
         if (!Boolean.TRUE.equals(table.getAttribute(Table.ADD_ONLY_TABLE_ATTRIBUTE))
-                && !Boolean.TRUE.equals(table.getAttribute(Table.APPEND_ONLY_TABLE_ATTRIBUTE))) {
-            throw new IllegalArgumentException("Argument table is neither add-only nor append-only");
+                && !Boolean.TRUE.equals(table.getAttribute(APPEND_ONLY_TABLE_ATTRIBUTE))) {
+            addOnlyTable = table.withAttributes(Map.of(Table.ADD_ONLY_TABLE_ATTRIBUTE, TRUE));
```

Comment:
Either use the qualified name everywhere, or nowhere. I prefer not to statically import `TRUE`.
```suggestion
            addOnlyTable = table.withAttributes(Map.of(Table.ADD_ONLY_TABLE_ATTRIBUTE, Boolean.TRUE));
```